schemas/com.mailgun/message_bounced/jsonschema/1-0-0

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Mailgun message bounced event",
+  "self": {
+    "vendor": "com.mailgun",
+    "name": "message_bounced",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "attachment-count": {
+      "description": "Number of attachments in attachment-x (where x is a number) fields.",
+      "type": "integer",
+      "maximum": 32767
+    },
+    "body-plain": {
+      "description": "text/plain body of the message.",
+      "type": "string",
+      "maxLength": 524288
+    },
+    "event": {
+      "description": "Event name ('bounced').",
+      "type": "string",
+      "maxLength": 16
+    },
+    "recipient": {
+      "description": "Recipient who could not be reached.",
+      "type": "string",
+      "maxLength": 512
+    },
+    "domain": {
+      "description": "Domain that sent the original message.",
+      "type": "string",
+      "maxLength": 256
+    },
+    "message-headers": {
+      "description": "String list of all MIME headers of the original message dumped to a JSON string (order of headers preserved).",
+      "type": "string"
+    },
+    "code": {
+      "description": "SMTP bounce error code in form (X.X.X).",
+      "type": "integer",
+      "maximum": 32767
+    },
+    "error": {
+      "description": "SMTP bounce error string.",
+      "type": "string"
+    },
+    "notification": {
+      "description": "Detailed reason for bouncing (optional).",
+      "type": "string"
+    },
+    "campaign-id": {
+      "description": "The id of campaign triggering the event.",
+      "type": "string"
+    },
+    "campaign-name": {
+      "description": "The name of campaign triggering the event.",
+      "type": "string"
+    },
+    "tag": {
+      "description": "Message tag, if it was tagged. See Tagging.",
+      "type": "string"
+    },
+    "mailing-list": {
+      "description": "The address of mailing list the original message was sent to.",
+      "type": "string"
+    },
+    "Message-Id": {
+      "description": "The Id of the message that bounced.",
+      "type": "string",
+      "maxLength": 1024
+    },
+    "timestamp": {
+      "description": "Number of seconds passed since January 1, 1970 (see securing webhooks).",
+      "type": "integer",
+      "maximum": 2147483647
+    },
+    "token": {
+      "description": "Randomly generated string with length 50 (see securing webhooks).",
+      "type": "string",
+      "maxLength": 64
+    },
+    "signature": {
+      "description": "String with hexadecimal digits generate by HMAC algorithm (see securing webhooks).",
+      "type": "string",
+      "pattern": "^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)$",
```

Comment:
This is quite unreliable `base64` regexp. Description doesn't look like Base64.

Reply:
Agreed. I am not sure why this was included.

Reply:
To be clear HMAC is often base64 encoded (and in this case it is), but I am not sure we need to check that here, rather than a proper validation (documented under securing webhooks). The question is do we need to check that here?

Reply:
Most likely because Schema Guru tries to recognize `base64`, but it inevitably results in false positives, especially for low-volume data.